ultimarc/devices/mini_pac.py

```diff
@@ -325,7 +325,7 @@ def _create_message_(self, config_file, cur_device_config=None):
                     if len(macro.action) > 0:
                         # Set the start point of the new macro
                         data.bytes[cur_position] = cur_macro
-                        macro_dict[macro.name.upper()] = cur_macro
```

Comment:
this name variable shouldn't be changed to product_name.

Reply:
Fixed.